src/App.tsx

```diff
@@ -1,61 +1,122 @@
+import React, { useState } from 'react';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+import { Todos } from './components/Types/Todo';
+import { getUserById } from './components/utils/getUserById';
+import { generalNewId } from './components/utils/generateNewld';
+import { TodoList } from './components/TodoList';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+const initialTodos: Todos = todosFromServer.map((todo) => ({
+  ...todo,
+  user: getUserById(todo.userId),
+}));
+
+export const App: React.FC = () => {
+  const [title, setTitle] = useState<string>('');
+  const [hasTitleIdError, setHasTitleError] = useState<boolean>(false);
```

Comment:
```suggestion
  const [title, setTitle] = useState('');
  const [hasTitleIdError, setHasTitleError] = useState(false);
```
it's redundant to specify primitive types